cassandra-diagnostics-core/src/main/java/io/smartcat/cassandra/diagnostics/module/slowquery/SlowQueryLogDecider.java

```diff
@@ -50,8 +50,7 @@ public boolean isForReporting(Query query) {
     private boolean executionTimeForLogging(long executionTimeInMilliseconds) {
         logger.trace("Checking if execution time:{} is above threshold: {}", executionTimeInMilliseconds,
                 slowQueryConfiguration.slowQueryThreshold());
-        if (slowQueryConfiguration.logAllQueries()
-                || executionTimeInMilliseconds >= slowQueryConfiguration.slowQueryThreshold()) {
+        if (executionTimeInMilliseconds > slowQueryConfiguration.slowQueryThreshold()) {
```

Comment:
I would suggest handling zero threshold case separately. So if it is zero, log everything (in accordance with the table name filter), if greater than zero, test with '>'.